Symportal terms 

|                    |                                                                                                                                                                                                                                  |
| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Pre-MED sequences  | Fasta file containing sequences from published samples that have been through the SymPortal quality control pipeline but have not yet been through Minimum Entropy Decomposition. 3,591,132 sequences. Last updated: 2024-02-13. |
| Post-MED sequences | Fasta file containing sequences from published samples that have been through the SymPortal quality control pipeline and Minimum Entropy Decomposition. 72,040 sequences. Last updated: 2024-02-13.                              |

[Minimum Entropy Decomposition (MED)](https://doi.org/10.1038/ismej.2014.195) provides a computationally efficient means to partition marker gene datasets into ‘MED nodes’, which represent homogeneous operational taxonomic units. By employing Shannon entropy, MED uses only the information-rich nucleotide positions across reads and iteratively partitions large datasets while omitting stochastic variation. When applied to analyses of microbiomes from two deep-sea cryptic sponges _Hexadella dedritifera_ and _Hexadella_ _cf_. _dedritifera_, MED resolved a key Gammaproteobacteria cluster into multiple MED nodes that are specific to different sponges, and revealed that these closely related sympatric sponge species maintain distinct microbial communities. MED analysis of a previously published human oral microbiome dataset also revealed that taxa separated by less than 1% sequence variation distributed to distinct niches in the oral cavity. The information theory-guided decomposition process behind the MED algorithm enables sensitive discrimination of closely related organisms in marker gene amplicon datasets without relying on extensive computational heuristics and user supervision.