Goals: 
-  goals for 16S and ITS2
	- see if the 3:1 molarity worked
	- how to separate the 16S and ITS2 when I used same indexing primers 
	- ~~cutadapt with the primers~~ 

sampling 
Sankey graphs- immune timepoints, even though fewer corals 

Nov 2-7
- [ ] Date_PCR and number of reads relationship??
- [ ] cut the n's off 
	- just bioinformatically cut the n's out
	- cutadapt to cut off everything before the adapter 
	- sequencer demulipexed- cut off indexing sequencing adapter
	- take just the 16S primer, cut off anything before that 
- [ ] look at what is different in both pasts 
	- is there overlap in asvs (venndiagram for 2 coral @ 2 timepoints)
	- exploratory statistics, im gonna add more samples
	- what are the microbes that are staying the same through time?
		- individuals?
		- species?

Oct 14-18
- [x] apply for tech position in Lisa's lab
	- [x] add to networking drive
- [x] jade pcr (tues)
- [x] art for poster
- [x] scratch3 extension
- [x] start CERF presentation
- [x] trying to map out nested clusters (wed)
	- figured out that hierarchical clusters won't work for my results bc there is overlapping in the nesting
	- switch gears to OHC
- [ ] bio retreat poster
- [x] remove "Date_16S " run analysis that way
- [ ] patterns with Date_16S?? 2_25 and 3_6
	- amount of reads?
	- Taxomony?? order, phylum
	- Run DEseq2 with df of cluster 1 and cluster 2 
			- LOOKS for diff in asv based on cluster 1 and cluster 2
			- don't need nmds info there 
	- new df by species
		- top 100 of asvs
		- plots by species 
		- facet by colony 
		- family 


Oct 5-10
- [x] submit Fulbright (mon)
- [x] help branae with bead cleanup (mon)
- [x] submit UCSB job app (tues)
- [x] book CERF room with noemie (tues)
- [x] fill out Bernhardt interest form (Wed)
- [x] phyloseq on samples >10K reads (tues)
- [x] networking spreadsheet
- [x] reread notes 
	- from vegan course
	- your 16S code
- [x] analyze my code in different ways 
	- bray-curtis, PCoA, CCA
	- run default for alpha diversity instead of choosing shannon and simpson
- [x] start poster for bio retreat

Sept 29-Oct 3
- [ ] check on 062019_T2_18_MCAV, did I use enriched?
Sept 22-26
- [ ] run entire dada2 pipeline
	- unity office hours changing CPU to get it to run more efficiently
	- running dada2 with big data
- [ ] packing day (thurs)
- [ ] try to read other files in jupyter notebook (fri)
- [ ] connect to phyloseq (fri)
- [ ] screenshot beckman coulter put in 16S protocol
- [ ] update protocols in lab github

Sept 15-19
- [x] multiqc on split_reads
	- its not cherry picking to choose the one with most number of reads
	- run on different cutadapt outputs, to see quality before dada2
- [x] separate R1 & R2!!!! in multiqc to set truncation length
	- [x] sep dir for r1 and r2
	- truncation length set to 250,230


Sept 8-12
- [x] sign up for CERF zoom webinars
- [x] 15 PCR (wed)
- [x] 15 (thurs)
- [x] add to funding spreadsheet in lab drive
- [x] col prep rna later tubes
- [x] save output folder onto github
- [x] wdhof follow up report
- [x] look at number of sequences/reads i get compared to double banding (https://www.quora.com/What-correlation-do-I-use-between-yes-no-maybe-data-and-a-scale-data)
- [ ] *do the sequences start with 16S primer? where did the barcodes go?*
	- *~~1. run dada2 on untrimmed 16S~~*
	- *2 run dada2 on more agressive trimming same filt*
	- *3. more agressive trimming (same as step 2) less agressive filtering in dada2*
FULBRIGHT
- [x] fill out teresas letter of rec form (mon night)
- [x] write letter of affiliation
- [x] check in with Maria Matz for FLE letter
- [x] prep for interview (fri 2-3 pm in oleary)

Sept 1-5
- [x] look over tristin IDP
- [x] look over branae idp
- [x] email recommenders 
- [x] write letter of affiliation
- [x] fulbright practice interview?