Goals: 
-  goals for 16S and ITS2
	- see if the 3:1 molarity worked
	- how to separate the 16S and ITS2 when I used same indexing primers 
	- ~~cutadapt with the primers~~ 

Sept 22-26
- [ ] run entire dada2 pipeline
	- unity office hours changing CPU to get it to run more efficiently
	- running dada2 with big data
- [ ] connect to phyloseq (wed)
- [ ] packing day (thurs)

Sept 15-19
- [x] multiqc on split_reads
	- its not cherry picking to choose the one with most number of reads
	- run on different cutadapt outputs, to see quality before dada2
- [x] separate R1 & R2!!!! in multiqc to set truncation length
	- [x] sep dir for r1 and r2
	- truncation length set to 250,230


Sept 8-12
- [x] sign up for CERF zoom webinars
- [x] 15 PCR (wed)
- [x] 15 (thurs)
- [x] add to funding spreadsheet in lab drive
- [x] col prep rna later tubes
- [x] save output folder onto github
- [x] wdhof follow up report
- [x] look at number of sequences/reads i get compared to double banding (https://www.quora.com/What-correlation-do-I-use-between-yes-no-maybe-data-and-a-scale-data)
- [ ] *do the sequences start with 16S primer? where did the barcodes go?*
	- *~~1. run dada2 on untrimmed 16S~~*
	- *2 run dada2 on more agressive trimming same filt*
	- *3. more agressive trimming (same as step 2) less agressive filtering in dada2*
FULBRIGHT
- [x] fill out teresas letter of rec form (mon night)
- [x] write letter of affiliation
- [x] check in with Maria Matz for FLE letter
- [x] prep for interview (fri 2-3 pm in oleary)

Sept 1-5
- [x] look over tristin IDP
- [x] look over branae idp
- [x] email recommenders 
- [x] write letter of affiliation
- [x] fulbright practice interview?