#### Used B14 barcoding primers

| Group: | 0   | i7 -                        | i7 -                        | i7 -                       | i7 -                        | i7 -                        | i7 -                        | i7 -                        | i7 -                        | i7 -                        | i7 -                        | i7 -                        | i7 -                        |
| ------ | --- | --------------------------- | --------------------------- | -------------------------- | --------------------------- | --------------------------- | --------------------------- | --------------------------- | --------------------------- | --------------------------- | --------------------------- | --------------------------- | --------------------------- |
|        |     | 1                           | 2                           | 3                          | 4                           | 5                           | 6                           | 7                           | 8                           | 9                           | 10                          | 11                          | 12                          |
| i5 -   | A   | 022024_BEL_CBC_T2_812_PAST  | 122022_BEL_CBC_T4_11_MCAV   | 022024_BEL_CBC_T3_847_PAST | 052022_BEL_CBC_T2_3_SSID    | 092023_BEL_CBC_T2_179_SSID  | 062024_BEL_CBC_T1_1350_PSTR | 122023_BEL_CBC_T3_523_MCAV  | 022024_BEL_CBC_T2_810_SSID  | 122023_BEL_CBC_T1_510_PSTR  | 042024_BEL_CBC_T2_1027_PAST | 012024_BEL_CBC_T2_599_MCAV  | 092023_BEL_CBC_T3_177_SSID  |
| i5 -   | B   | 112023_BEL_CBC_T1_274_PAST  | 112023_BEL_CBC_T3_357_OFAV  | 052022_BEL_CBC_T2_12_MCAV  | 112023_BEL_CBC_T3_354_PSTR  | 042024_BEL_CBC_T4_1042_OFAV | 082024_BEL_CBC_T3_1553_MCAV | 122022_BEL_CBC_T3_116_PAST  | 092023_BEL_CBC_T4_95_OFAV   | 022024_BEL_CBC_T2_815_OFAV  | 052022_BEL_CBC_T1_1_PAST    | 122023_BEL_CBC_T3_528_PAST  | 062024_BEL_CBC_T4_1473_PSTR |
| i5 -   | C   | 092023_BEL_CBC_T3_174_OFAV  | 092023_BEL_CBC_T2_183_MCAV  | 012024_BEL_CBC_T1_559_MCAV | 082024_BEL_CBC_T3_1557_PAST | 112023_BEL_CBC_T3_353_MCAV  | 122022_BEL_CBC_T3_151_PSTR  | 022024_BEL_CBC_T4_873_MCAV  | 112023_BEL_CBC_T4_398_PSTR  | 062024_BEL_CBC_T3_1424_PAST | 092023_BEL_CBC_T1_173_SSID  | 052022_BEL_CBC_T3_1_MCAV    | 062019_BEL_CBC_T1_5_SSID    |
| i5 -   | D   | 062019_BEL_CBC_T2_17_SSID   | 022024_BEL_CBC_T2_805_SSID  | 112023_BEL_CBC_T4_397_PSTR | 122022_BEL_CBC_T2_92_MCAV   | 092023_BEL_CBC_T4_96_MCAV   | 042024_BEL_CBC_T2_1034_SSID | 122023_BEL_CBC_T1_506_PAST  | 122022_BEL_CBC_T1_158_PAST  | 112023_BEL_CBC_T2_305_PAST  | 022024_BEL_CBC_T2_811_PAST  | 062024_BEL_CBC_T3_1427_MCAV | 122022_BEL_CBC_T4_1_OFAV    |
| i5 -   | E   | 022024_BEL_CBC_T4_870_MCAV  | 042024_BEL_CBC_T3_946_OFAV  | 042024_BEL_CBC_T1_928_OANN | 062019_BEL_CBC_T2_21_PAST   | 062019_BEL_CBC_T3_5_SSID    | 122023_BEL_CBC_T2_512_OFAV  | 112023_BEL_CBC_T2_306_MCAV  | 122023_BEL_CBC_T3_533_OFAV  | 022024_BEL_CBC_T2_809_MCAV  | 082024_BEL_CBC_T3_1556_OFAV | 112023_BEL_CBC_T1_275_SSID  | 112023_BEL_CBC_T2_314_SSID  |
| i5 -   | F   | 042024_BEL_CBC_T4_1044_PSTR | 082024_BEL_CBC_T2_1508_SSID | 122022_BEL_CBC_T1_156_SSID | 042024_BEL_CBC_T3_955_OFAV  | 112023_BEL_CBC_T3_359_PAST  | 122023_BEL_CBC_T4_537_PSTR  | 042024_BEL_CBC_T4_1045_MCAV | 022024_BEL_CBC_T3_854_PAST  | 042024_BEL_CBC_T3_944_SSID  | 092023_BEL_CBC_T3_175_MCAV  | 012024_BEL_CBC_T2_595_PAST  | 052022_BEL_CBC_T3_15_PSTR   |
| i5 -   | G   | 022024_BEL_CBC_T3_848_MCAV  | 082024_BEL_CBC_T1_1481_PSTR | 012024_BEL_CBC_T4_669_MCAV | 122022_BEL_CBC_T1_150_PAST  | 042024_BEL_CBC_T4_1041_MCAV | 062024_BEL_CBC_T3_1436_MCAV | 022024_BEL_CBC_T2_806_SSID  | 082024_BEL_CBC_T2_1512_PAST | 122023_BEL_CBC_T3_529_MCAV  | 022024_BEL_CBC_T2_813_MCAV  | 092023_BEL_CBC_T3_182_MCAV  | 092023_BEL_CBC_T1_174_MCAV  |
| i5 -   | H   | 122023_BEL_CBC_T4_540_MCAV  | 012024_BEL_CBC_T2_591_PAST  | 012024_BEL_CBC_T4_665_MCAV | 112023_BEL_CBC_T2_313_SSID  | 052022_BEL_CBC_T2_5_PAST    | 022024_BEL_CBC_T3_844_OFAV  | 092023_BEL_CBC_T2_175_PAST  | 062019_BEL_CBC_T2_23_MCAV   | 122023_BEL_CBC_T4_535_MCAV  | 082024_BEL_CBC_T2_1511_SSID | 122022_BEL_CBC_T2_87_OFAV   | Negative                    |
|        |     | 1-8                         | 9-16                        | 17-24                      | 25-32                       | 33-40                       | 41-48                       |                             | 57-64                       |                             | 73-80                       |                             | 89-96                       |

## Protocol
#### IV. Indexing PCR 
- each sample is indexed with a unique combination of i5 and i7 primers 

| Reagent     | Amount per 1 rxn (uL) | MasterMix Amount (uL) + 10% | Divided by two (if needed) |
| ----------- | --------------------- | --------------------------- | -------------------------- |
| Buffer      | 5                     | 528                         | 264                        |
| dNTP (10mM) | 0.5                   | 52.8                        | 26.4                       |
| DNA         |                       |                             | 0                          |
| Polymerase  | 0.25                  | 26.4                        | 13.2                       |
| Water       | 16.25                 | 1716                        | 858                        |
| Total       | 22                    | 2323.2                      | 1161.6                     |
using HotStart reagents for this protocol using the following combination of reagents:
- 1uL of pooled product as template in the indexing PCR
- indexing PCR done in 25uL total reaction volume using KAPA Ready Mix

**Steps:**
1. Use strip tubes to create a master mix for each i5 and i7 primer
	
| Index master mix rows (vertical) |           |                            |
| -------------------------------- | --------- | -------------------------- |
| Reagent                          | MM amount | Divided by two (if needed) |
| Big master mix                   | 145.2     | 72.6                       |
| N50xx index                      | 13.2      | 6.6                        |
| Total                            | 158.4     | 79.2                       |

| Index master mix columns (horizontal) |                       |                            |
| ------------------------------------- | --------------------- | -------------------------- |
| Reagent                               | Amount per 1 rxn (uL) | Divided by two (if needed) |
| Big master mix                        | 96.8                  | 48.4                       |
| N50xx index                           | 8.8                   | 4.4                        |
| Total                                 | 105.6                 | 52.8                       |
2. use multi-channel pipette to dispense the master mixes into appropriate tube
	1. add 12uL of each i5 and i7 master mix to each reaction tube - following the template to track the combination of barcodes used for each sample
	2. 12uL i5 master mix + 12uL i7 master mix + 1uL template = 25uL total reaction
3. add DNA to individual reactions 
4. run thermocycler program

Thermocycler program:
1. 98 for 30 sec
2. **98 for 10 sec**
3. **67 for 30 sec**
4. **72 for 20 sec** 
*repeat 2-4 for 7 cycles (# of cycles varies depending on input)*
5. 72 for 2 min
#### V. Gel electrophoresis
perform steps as above with one addition:
- for each row in the gel, **include both a ladder and a non-indexed control**
- pre- and post-indexed samples should be different sizes
- **1uL product used to load gel to check indexing**


#### **VI. Purification with ampure beads**
obtain, label, and cross-link new strip tubes
start with the manufacturer protocol using 1.8X-1.0X bead ratio and 10uL-25uL PCR product
- ratio of beads will change the size you select for 
 *may need to re-clean samples if gel images show that multiple bands were not removed*
- 1.0x will get rid of <200 bp dimers, 1.8X will get rid of dimer <100 bp
https://www.beckman.com/reagents/genomic/cleanup-and-size-selection/pcr/bead-ratio
![[bead_SizeSelection.png]] 

- **NOTE:** all calculations can be done here https://docs.google.com/spreadsheets/d/1O_NJCFvnBztKm_G88Sx-gEKD7CwR44iEaRjyxS_N32E/edit?gid=1947158502#gid=1947158502
1.  make fresh 80% ethanol in a 50mL tube (label and parafilm when not in use)
	- paste filled out table here

| Number of Samples | 80% EtOH for each sample (uL) | Total 80% EtoH needed (uL) | Volume 100% EtOH (uL) | Volume H2O (uL) |
| ----------------- | ----------------------------- | -------------------------- | --------------------- | --------------- |
| 96                | 360                           | 34560                      | 27648                 | 6912            |
|                   |                               | in mL                      |                       |                 |
|                   |                               | 34.56                      | 27.648                | 6.912           |


2. Determine whether or not a plate transfer is necessary. If the PCR reaction volume multiplied by 2.8 exceeds the volume of the PCR plate, a transfer to larger tubes is required.
3. Gently shake the Clean NGS Mag PCR Clean-up aliquot to resuspend any Magnetic particles that may have settled. 
	1. Add CleanNGS Mag PCR Clean-up volume table below: 
	
| Bead Concentration | PCR volume (uL) | Added beads volume (uL) | Total # Samples | Total Bead Volume (uL) |
| ------------------ | --------------- | ----------------------- | --------------- | ---------------------- |
| 0.9                | 24              | 21.6                    | 96              | 2073.6                 |

**Note:** The volume of CleanNGS Mag PCR Clean-up for a given reaction can be determined from the following equation:  
	*(Volume of Mag Beads per reaction) = (Bead Concentration) x (PCR Reaction Volume)*
	
3. Mix reagent and PCR reaction thoroughly by pipette mixing 5 times.
4. Incubate the mixed samples for 5 minutes at room temperature for maximum recovery. This step allows the binding of PCR products 300bp (based on concentration) and greater to the Magnetic beads.  After mixing, the color of the mixture should appear homogenous.
5. Place the reaction plate onto a 96 well Magnet Plate for 3 minutes or wait until the solution is clear.  Wait until the solution is clear before proceeding to the next washing step. Otherwise there may be beads loss.
6. Aspirate the cleared solution from the reaction plate and discard This step must be performed while the reaction plate is placed on the 96 magnetic plate. Avoid disturbing the settled magnetic beads. If beads are drawn into tips, leave behind a few microliters of solution.
7. Dispense **180 uL of 80% ethanol** to each well of the reaction plate and incubate for **1 min** at room temperature.  Aspirate out the ethanol and discard.  Repeat for a total of two washes.  It is important to perform these steps with the reaction plate on a 96 well Magnetic Plate. Do not disturb the settled Magnetic beads. 
	1. Remove all of the ethanol from the bottom of the well to avoid ethanol carryover. **Bump pipette tip up to 200 uL, may need to use p20 multichannel** 
	2. NOTE: A 5 min air dry at room temperature is recommended for the evaporation of the remaining traces of ethanol. **Do not overdry the beads** (the layer of settled beads appears cracked) as this will significantly decrease elution efficiency.
8. Take off the plate from the Magnetic plate, add equal amount of original volume or less of elution buffer (Reagent grade water, TRIS-HCl pH 8.0, or TE buffer) to each well of the reaction plate and pipette mix 5 times. 
	1. mix until homogeneous and there are no beads on tube wall
9. Incubate at room temperature for 10 minutes. 
10. Place the plate on a magnetic separation device to magnetize the CleanNGS particles. Incubate at room temperature until the CleanNGS particles are completely cleared from solution. 
11. Transfer the cleared supernatant containing purified DNA and/or RNA to a new (RNase-free) 96-well microplate and seal with non-permeable sealing film. 
12. Store the plate at 2-8°C if storage is only for a few days. For long-term storage, samples should be kept at -20°C.

#### VII. Gel Electrophoresis 
1. Refer to steps II and V
2. Run a gel with select samples to confirm bead size selection worked 
#### VIII. Sequencing Prep
1. Assess need for Gel Extraction
	1. If need gel extraction refer to [[Qiagen Gel Extraction]]	
	2. If no need, skip to step 2
2. Qubit each indexed sample
3. Pool all indexed samples according to final concentration
4. Calculate Molarity by plugging in concentrations, volume 
https://docs.google.com/spreadsheets/d/1O_NJCFvnBztKm_G88Sx-gEKD7CwR44iEaRjyxS_N32E/edit?gid=1110196762#gid=1110196762
5. Fill out necessary sequencing paperwork